src/extension.ts

```diff
@@ -1,9 +1,40 @@
 import { ExtensionContext } from "vscode";
 import { GodotTools } from "./godot-tools";
+import * as path from "path";
+import * as vscode from 'vscode'
 
 
 let tools: GodotTools = null;
 
+export class DocContent {
+	static docItems: vscode.QuickPickItem[] = []
+	static builtIns = new Set()
+	static dataDirectory
+
+	constructor(dataPath : string) {
+		DocContent.dataDirectory = path.join(dataPath, 'godot', 'doc')
+		let indexLocation = path.join(dataPath, 'godot', 'doc', `index.json`)
+
+		require('fs').readFile(indexLocation, 'utf8', function (err, data) {
```

Comment:
Not much experience with ts, would `import { readFile } from 'fs';` work?   it may be better to keep the import consistent